Dockerfile

```diff
@@ -0,0 +1,17 @@
+FROM php:7.4-cli
```

Comment:
Nice. Long time I don't check this repo.
Just make sure in case a php version goes unsupported (as in: no more security updates) it's a good idea to also raise the version to something secure and also test it with the latest version.
If we test the higher and lower limits we can be sure all versions in between will work.
That way in case a merchant is using an older version of PHP that has known CVEs we notify them (or ask their rep in merchant services to do so) so that they upgrade, no matter what the team lead says about supporting old versions! In case a team lead insists make them sign a document stating they will pay for the damages in case the merchant gets hacked, thank you! XD